byway/src/main/webapp/home.js

```diff
@@ -44,7 +44,7 @@ function loadPastTrip(){
     tripIds.forEach(trip => {
       let isDestinationsMissing = trip.destinations.length == 0;
       let isInterestsMissing = trip.interests.length == 0;
-      (isDestinationsMissing|| isInterestsMissing) ? showIncompleteTrip(tripNum, trip, isDestinationsMissing, isInterestsMissing) : showCompleteTrip(tripNum, trip);
+      (isDestinationsMissing|| isInterestsMissing) ? showIncompleteTrip(tripNum, trip, isDestinationsMissing, isInterestsMissing) : showCompleteTrip(trip);
```

Comment:
I know that this is not really part of this change, but can you please update this ternary operation into an if statement. Ternary operations are usually only used for assignment. An if statement would make this code more clear.

Reply:
done